pkg/orchestrator/containers.go

```diff
@@ -0,0 +1,97 @@
+package orchestrator
```

Comment:
if we're going to put together all functions in a single file in `mysql` package, let's be consistent and do the same for `orchestrator`